static/js/actions/index.js

```diff
@@ -132,3 +132,12 @@ export function fetchDashboard() {
       });
   };
 }
+
+
+// dashboard expander actions
+export const TOGGLE_DASHBOARD_EXPANDER = 'TOGGLE_DASHBOARD_EXPANDER';
+
+export const toggleDashboardExpander = (courseId, newValue) => ({
+  type: TOGGLE_DASHBOARD_EXPANDER,
+  payload: { courseId, newValue }
+});
```

Comment:
do you think it might make sense to stick these actions in with the `ui` actions? I had imagined the `ui` top level key on the store being for storing miscellaneous pieces of UI state (dialog open / closed, etc).